tests/ext/test_jobqueue.py

```diff
@@ -85,6 +85,23 @@ class TestJobQueue:
         " We recommend double checking if the passed value is correct."
     )
 
+    async def test_repr(self, app):
+        import datetime as dtm
```

Comment:
this import is already defined at the top?

Reply:
Yep. Removed